server/protocols/jmap-rfc-8621/src/test/scala/org/apache/james/jmap/json/SessionSerializationTest.scala

```diff
@@ -87,28 +85,32 @@ object SessionSerializationTest {
   private val IS_NOT_PERSONAL : IsPersonal = IsPersonal(false)
   private val IS_NOT_READ_ONLY : IsReadOnly = IsReadOnly(false)
 
-  private val ACCOUNT_1 = Account(
+  private val ACCOUNT_1: Account = Account.from(
     name = USER_1,
     isPersonal = IS_PERSONAL,
     isReadOnly = IS_NOT_READ_ONLY,
-    accountCapabilities = Set(CORE_CAPABILITY))
-  private val ACCOUNT_2 = Account(
+    accountCapabilities = Set(CORE_CAPABILITY)) match {
+      case Left(ex: IllegalArgumentException) => throw ex
+      case Right(account: Account) => account
+    }
+
+  private val ACCOUNT_2: Account = Account.from(
     name = USER_2,
     isPersonal = IS_NOT_PERSONAL,
     isReadOnly = IS_NOT_READ_ONLY,
-    accountCapabilities = Set(CORE_CAPABILITY))
-  private val ACCOUNTS = Map(
-    USER_1_ID -> ACCOUNT_1,
-    USER_2_ID -> ACCOUNT_2,
-  )
+    accountCapabilities = Set(CORE_CAPABILITY))  match {
+      case Left(ex: IllegalArgumentException) => throw ex
```

Comment:
isn't what `.get` is doing?

Reply:
I can't find `get()`, there is `getOrElse(anotherAccount: Account)` but I cannot use it as the problem is `how to create an Account`

Reply:
you can do  something like : `.right.get` no ?

Reply:
`.right` is deprecated:
```
@deprecated("Either is now right-biased, use methods directly on Either", "2.13.0")
```

Reply:
If you want something shorter in the tests you can do : `.toOption.get`

Reply:
thanks!